### ISSUE TITLE
Please include Privacy Menifest File ASAP

### ISSUE THREAD

**Issue by ashurajput1** [relatedcode/ProgressHUD#166]

**Comment by relatedcode** [comment]
@ashurajput1 Give me a clear instruction ASAP.

**Comment by ashurajput1** [comment]
sir the privacy menifest file is not included in your code . I am using your library .if apple don't find the privacy menifest file for progrees hud then our app can be rejected.

**Comment by ashurajput1** [comment]
> @ashurajput1 Give me a clear instruction ASAP.



> @ashurajput1 Give me a clear instruction ASAP.
sir the privacy menifest file is not included in your code . I am using your library .if apple don't find the privacy menifest file for progrees hud then our app can be rejected.

**Comment by relatedcode** [comment]
Yes, I know about the problem. Give me what file I need to include. I will compose the file content by myself. However, I don't have the time to figure out the details right now...
